src/main/java/org/camunda/community/benchmarks/config/AsyncConfiguration.java

```diff
@@ -56,4 +57,14 @@ public TaskScheduler taskScheduler() {
         scheduler.initialize();
         return scheduler;
     }
+
+    @Bean
+    public ScheduledExecutorService scheduledExecutorService() {
+        LOGGER.debug("Creating Async Task Scheduler");
+        ThreadPoolTaskScheduler scheduler  = new ThreadPoolTaskScheduler();
+        scheduler.setPoolSize(corePoolSize);
+        scheduler.setThreadNamePrefix("ThreadPoolTaskScheduler-");
+        scheduler.initialize();
+        return scheduler.getScheduledExecutor();
+    }

```

Comment:
Why is this needed if process instances and message scenarios were already working?

Reply:
Is this due to changes in Spring Zeebe 8.2 vs 8.1? If so than it would be an issue for the entire project and not just BPMN.

Reply:
If it's really needed, it should at least have a different `ThreadNamePrefix`

Reply:
I feel uneasy with having three different thread pools